lib/typelizer/inline_type.rb

```diff
@@ -0,0 +1,22 @@
+module Typelizer
+  # InlineType is mainly used with Alba plugin to represent inline associations.
+  # `name `method` is the same interface as `Interface` class.
+  class InlineType
+    TEMPLATE = <<~ERB.strip
+      {
+        <%- properties.each do |property| %>
+          <%= property %>;
+        <% end %>
+      }
+    ERB
+    def initialize(serializer:, config:)
+      @serializer = serializer
+      @config = config
+    end
+
+    def name
+      properties = SerializerPlugins::Alba.new(serializer: @serializer, config: @config).properties
```

Comment:
This is kind of dirty hack, but it's recursive so that inline associations can be deeply nested.